### ISSUE TITLE
:gift: :broom: Hyrax: Get double_combo merged into main

### ISSUE THREAD

**Issue by ShanaLMoore** [scientist-softserv/hykuup_knapsack#127]
# Story

double_combo is Rob's branch that is currently being used to support a valkyrized hyrax for GBH. We've been developing on it as well and ideally need to get it merged into Hyrax main. 

## Prerequisite Tasks
- [x] https://github.com/scientist-softserv/hykuup_knapsack/issues/199
- [x] get build green

# Acceptance Criteria

- [ ] double_combo has been reviewed and approved by the maintainers of Hyrax

# Screenshots / Video

<details>
<summary></summary>

</details>

# Testing Instructions and Sample Files

-

# Notes

Is there, or when will there be, a Hyrax v5.0.0 release? We should point to that instead of the release candidate version.

**Comment by jillpe** [comment]
blocked by derivatives issues

**Comment by ShanaLMoore** [comment]
merged 🎉
